src/Components/Login/Login.module.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../style/variables.scss';
+
```

Comment:
Love the styling, you nailed it!

Reply:
Agreed, this looks super clean and `profresh`